src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/SymbolMatcherTests.cs

```diff
@@ -769,7 +769,7 @@ public class C
     public delegate (int, int) F();
 }";
             var source1 = @"
-public struct C
```

Comment:
Oh, yes. Sorry I missed that.

---
In reply to: [116845747](https://github.com/dotnet/roslyn/pull/19283#discussion_r116845747) [](ancestors = 116845747,116844195)